src/components/DisplayEmbedTag/DisplayEmbedTag.jsx

```diff
@@ -11,15 +11,28 @@ import PropTypes from 'prop-types';
 import DisplayImageTag from './DisplayImageTag';
 import DisplayVideoTag from './DisplayVideoTag';
 
-const DisplayEmbedTag = ({ embedTag, className }) => {
+const DisplayEmbedTag = ({ embedTag, className, deletedOnSave }) => {
```

Comment:
Tror vi burde vente her. Denne skal ikke brukes i editoren. Har opprettet egne slate felter for dette.